### PR TITLE
Following our discussion, improving OCamlPro's visibility a little bit.

### PR DESCRIPTION
**Pull request by AltGr**
(PRing again to master as the PR to 'redesign' was closed when the branch was deleted)

ref: #304, #308

### CODE REVIEW

**Comment by AltGr** [comment]
@amirmc, @avsm, @Chris00, @dbuenzli, hopefully this updates fixes the concerns commented earlier.

File: site/community/support.md [hunk 1]
```diff
@@ -8,20 +8,27 @@ A great way to get free support is by using the active [mailing
...
-OCamlPro is a member of the OCaml Consortium.
-
-[Gerd Stolpmann](http://www.gerd-stolpmann.de/buero/work_ocaml_search.html.en)<br />
-Gerd Stolpmann has been helping companies master OCaml since 2005. He is
+### <img src="/img/ocamlpro.png" alt="OCamlPro" style="height: 2ex; vertical-align: text-bottom" />
+
+[OCamlPro](http://www.ocamlpro.com/) is the creator of many open-source tools
+widely used throughout the community, such as
+[Try OCaml](http://try.ocamlpro.com/), the
+[OPAM package manager](http://opam.ocamlpro.com/) or
```
Comment (avsm) [inline site/community/support.md]: s/or/and

Comment (AltGr) [inline site/community/support.md]: Fixed, thanks

**Comment by agarwal** [comment]
Is this pull request now obsolete, given #304 was already merged?

**Comment by AltGr** [comment]
It hasn't: it was automatically closed by github when the "redesign" branch was deleted.

**Comment by pw374** [comment]
Are we still waiting for something before merging this PR?

**Comment by amirmc** [comment]
Sorry, I think that's my fault.  I'm happy with the PR though I'd make changes to the `community/index.md` page after merging.  @agarwal commented about the inline images but I don't think we clarified anything.  I'm ok with them as they look reasonable (we can revert to text-based links in the future if they turn out to be a problem).

@agarwal Are you ok with merging this?

**Comment by agarwal** [comment]
@amirmc Please do merge it. We can always tweak aesthetic issues later.

**Comment by amirmc** [comment]
This is merged but doesn't appear to be live. Is it just me?

**Comment by pw374** [comment]
It is live now. (But there's a deeper problem with the build script, I'm investigating.)

edit: I think it works okay now.

**Comment by agarwal** [comment]
FYI, I consider it a bug that pull requests get automatically closed if the target branch is deleted, so I reported the problem we had here to GitHub.

**Pull request merged**